src/geo_key_directory.rs

```diff
@@ -657,3 +662,10 @@ enum GeoKeyDirectoryTag {
     VerticalDatum = 4098,
     VerticalUnits = 4099,
 }
+
+#[derive(Debug, Clone, Copy, PartialEq, TryFromPrimitive, IntoPrimitive)]
+#[repr(u16)]
+pub enum RasterType {
+    RasterPixelIsArea = 1,
+    RasterPixelIsPoint = 2,
+}
```

Comment:
A couple of thoughts here:

1. Should we add enum variants for `RasterType=0` (undefined/unknown) and `RasterType=32767` (user-defined), even though they're not recommended at https://docs.ogc.org/is/19-008r4/19-008r4.html#_requirements_class_gtrastertypegeokey?
2. Could you add a link to the GeoTIFF spec's `GTRasterTypeGeoKey` requirements for this please

Reply:
Done.